crates/sui-e2e-tests/tests/per_epoch_config_stress_tests.rs

```diff
@@ -0,0 +1,187 @@
+    };
+    let handle2 = {
+        let test_env = test_env.clone();
+        tokio::spawn(async move { run_admin_thread(test_env, target_epoch, gas2.0).await })
+    };
+    tokio::try_join!(handle1, handle2).unwrap();
+}
+
+async fn run_admin_thread(test_env: Arc<TestEnv>, target_epoch: EpochId, gas_id: ObjectID) {
+    let deny_list_object_init_version = test_env
+        .get_latest_object_ref(&SUI_DENY_LIST_OBJECT_ID)
+        .await
+        .1;
+    loop {
+        let gas = test_env.get_latest_object_ref(&gas_id).await;
+        let deny: bool = random();
+        let tx_data = test_env
+            .test_cluster
+            .test_transaction_builder_with_gas_object(test_env.regulated_coin_owner, gas)
+            .await
+            .move_call(
+                SUI_FRAMEWORK_PACKAGE_ID,
+                "coin",
+                if deny {
+                    "deny_list_v2_add"
+                } else {
+                    "deny_list_v2_remove"
+                },
+                vec![
+                    CallArg::Object(ObjectArg::SharedObject {
+                        id: SUI_DENY_LIST_OBJECT_ID,
+                        initial_shared_version: deny_list_object_init_version,
+                        mutable: true,
+                    }),
+                    CallArg::Object(ObjectArg::ImmOrOwnedObject(
+                        test_env.get_latest_object_ref(&test_env.deny_cap_id).await,
+                    )),
+                    CallArg::Pure(bcs::to_bytes(&DENY_ADDRESS).unwrap()),
+                ],
+            )
+            .with_type_args(vec![test_env.regulated_coin_type.clone()])
+            .build();
+        let effects = test_env
+            .test_cluster
+            .sign_and_execute_transaction(&tx_data)
+            .await
+            .effects
+            .unwrap();
+        let executed_epoch = effects.executed_epoch();
+        if executed_epoch >= target_epoch {
+            break;
+        }
+    }
+}
+
+async fn run_transfer_thread(test_env: Arc<TestEnv>, target_epoch: EpochId, gas_id: ObjectID) {
+    loop {
+        let gas = test_env.get_latest_object_ref(&gas_id).await;
+        let tx_data = test_env
+            .test_cluster
+            .test_transaction_builder_with_gas_object(test_env.regulated_coin_owner, gas)
+            .await
+            .move_call(
+                SUI_FRAMEWORK_PACKAGE_ID,
+                "pay",
+                "split_and_transfer",
+                vec![
+                    CallArg::Object(ObjectArg::ImmOrOwnedObject(
+                        test_env
+                            .get_latest_object_ref(&test_env.regulated_coin_id)
+                            .await,
+                    )),
+                    CallArg::Pure(bcs::to_bytes(&1u64).unwrap()),
+                    CallArg::Pure(bcs::to_bytes(&DENY_ADDRESS).unwrap()),
+                ],
+            )
+            .with_type_args(vec![test_env.regulated_coin_type.clone()])
+            .build();
```

Comment:
Should we have some PTB related tests too? (i.e. transactions that do not use Move calls)